api/src/kots_app/kots_ffi.ts

```diff
@@ -751,7 +757,7 @@ export async function kotsRewriteImagesInVersion(app: KotsApp, downstreams: stri
     await statusServer.termination((resolve, reject, obj): boolean => {
       // Return true if completed
       if (obj.status === "running") {
-        Promise.all([stores.kotsAppStore.setImageRewriteStatus(obj.display_message, "running")]);
+        stores.kotsAppStore.setImageRewriteStatus(obj.display_message, "running");
```

Comment:
not sure, is not awaiting here intentional?

Reply:
irl'ed